Uno.Gallery/Uno.Gallery.Shared/Uno.Gallery.Shared.projitems

```diff
@@ -212,7 +212,7 @@
     <AndroidAsset Include="$(MSBuildThisFileDirectory)Assets\Animations\*.json" Link="Assets\Assets\Animations\%(FileName)%(Extension)" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="$(MSBuildThisFileDirectory)Assets\UnoLogo.svg" />
+    <Content Include="$(MSBuildThisFileDirectory)Assets\Uno-logo.svg" />
```

Comment:
There is already UnoLogo.svg file. While add a new Uno-logo.svg one?

Reply:
@Youssef1313 For some reason the Pascal Case didn't work for displaying the svg

Reply:
That shouldn't really happen :/

Reply:
Would you be able to double check? Preferrably after doing a "rebuild" just in case?

Reply:
@Youssef1313 So, I tested this and I can still reproduce it, when I use the `UnoLogo.svg` (with the svg code the same) I get this error:
```
[DOTNET] [41m[30mfail[39m[22m[49m: Uno.UI.Svg.SvgProvider[0]
[DOTNET]       Failed to load SVG image.
[DOTNET]       System.Xml.XmlException: Invalid character in the given encoding. Line 1, position 1.
[DOTNET]          at System.Xml.XmlTextReaderImpl.Throw(Exception e)
[DOTNET]          at System.Xml.XmlTextReaderImpl.Throw(String res, String arg)
[DOTNET]          at System.Xml.XmlTextReaderImpl.Throw(Int32 pos, String res)
[DOTNET]          at System.Xml.XmlTextReaderImpl.InvalidCharRecovery(Int32& bytesCount, Int32& charsCount)
[DOTNET]          at System.Xml.XmlTextReaderImpl.GetChars(Int32 maxCharsCount)
[DOTNET]          at System.Xml.XmlTextReaderImpl.ReadData()
[DOTNET]          at System.Xml.XmlTextReaderImpl.SwitchEncoding(Encoding newEncoding)
[DOTNET]          at System.Xml.XmlTextReaderImpl.SwitchEncodingToUTF8()
[DOTNET]          at System.Xml.XmlTextReaderImpl.ParseXmlDeclaration(Boolean isTextDecl)
[DOTNET]          at System.Xml.XmlTextReaderImpl.Read()
[DOTNET]          at System.Xml.XmlTextReader.Read()
[DOTNET]          at Svg.SvgTextReader.Read() in /_/externals/SVG/Source/SvgTextReader.cs:line 66
[DOTNET]          at Svg.SvgDocument.Create[SvgDocument](XmlReader reader) in /_/externals/SVG/Source/SvgDocument.cs:line 363
[DOTNET]          at Svg.SvgDocument.Open[SvgDocument](Stream stream, Dictionary`2 entities) in /_/externals/SVG/Source/SvgDocument.cs:line 299
[DOTNET]          at Svg.Model.SvgExtensions.Open(Stream stream) in /_/src/Svg.Model/SvgExtensions.IO.cs:line 329
[DOTNET]          at Svg.Skia.SKSvg.Load(Stream stream) in /_/src/Svg.Skia/SKSvg.Model.cs:line 63
[DOTNET]          at Uno.UI.Svg.SvgProvider.<>c__DisplayClass22_0.<LoadSvgAsync>b__0()
```

while with `Uno-logo.svg` it displays correctly
![image](https://github.com/unoplatform/Uno.Gallery/assets/78549750/6922d81d-46b5-45d7-867d-4e767eacb45b)

Reply:
I think this needs to be investigated further :/

Reply:
Also, what if you replace the old file with the new file and keeping the old name? Does it work?

Reply:
@Youssef1313 I essentially did that - I took the new file contents (because I changed a few things) and I copied them over to the old file, with the old name, still the same error 🤔

Reply:
I think we should at least open an issue to investigate further.